src/quantity_api/__init__.py

```diff
@@ -17,5 +17,26 @@ def value(self) -> V: ...
     def unit(self) -> U: ...
 
     ### Dunder Methods
-
     def __eq__[B](self, other: Self[op.CanEq[V, B], U], /) -> B: ...
+    def __ne__[B](self, other: Self[op.CanEq[V, B], U], /) -> B: ...
+    def __lt__[B](self, other: Self[op.CanOrd[V, B], U], /) -> B: ...
+    def __le__[B](self, other: Self[op.CanOrd[V, B], U], /) -> B: ...
+    def __gt__[B](self, other: Self[op.CanOrd[V, B], U], /) -> B: ...
+    def __ge__[B](self, other: Self[op.CanOrd[V, B], U], /) -> B: ...
```

Comment:
Should these be ...?

```suggestion
    def __ne__[B](self, other: Self[op.CanNe[V, B], U], /) -> B: ...
    def __lt__[B](self, other: Self[op.CanLt[V, B], U], /) -> B: ...
    def __le__[B](self, other: Self[op.CanLe[V, B], U], /) -> B: ...
    def __gt__[B](self, other: Self[op.CanGt[V, B], U], /) -> B: ...
    def __ge__[B](self, other: Self[op.CanGe[V, B], U], /) -> B: ...
```